src/ray/raylet/node_manager.h

```diff
@@ -139,9 +131,6 @@ class NodeManager {
   /// The lineage cache for the GCS object and task tables.
   LineageCache lineage_cache_;
   std::vector<ClientID> remote_clients_;
-  /// A set of all of the remote clients that have been removed. In principle,
-  /// this could grow unbounded.
-  std::unordered_set<ClientID> removed_clients_;
```

Comment:
I think I added another usage of `removed_clients_` in #2429, so probably will need to rebase and remove that.